tests/pytests/unit/beacons/test_ps.py

```diff
@@ -0,0 +1,76 @@
+"""
+    tests.pytests.unit.beacons.test_ps
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    ps usage beacon test cases
+"""
+import pytest
+import salt.beacons.ps as ps
+from tests.support.mock import patch
+
+
+class FakeProcess:
+    def __init__(self, _name, pid):
+        self._name = _name
+        self.pid = pid
+
+    def name(self):
+        return self._name
+
+
+@pytest.fixture
+def patch_opts():
```

Comment:
Personally, I wouldn't use a fixture for this and would just use pass these kwargs to `patch()`, but it's not a merge blocker.

Reply:
Fixed -- sorry, I got a bit too caught up in line-by-line translation!